.github/workflows/maven.yml

```diff
@@ -10,12 +10,32 @@ jobs:
       matrix:
         java: [ '11' ]
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up JDK
-      uses: actions/setup-java@v2
-      with:
-        java-version: ${{ matrix.java }}
-        distribution: 'temurin'
-        cache: maven
-    - name: Build with Maven
-      run: mvn clean install -U
+      - uses: actions/checkout@v2
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          java-version: ${{ matrix.java }}
+          distribution: 'temurin'
+          cache: maven
+      - name: Build with Maven
+        run: mvn clean install -U
+
+  snyk:
```

Comment:
I'd break this out into it's own workflow as it and the build have no dependency on each other